akka-stream/src/main/scala/akka/stream/ActorFlowMaterializer.scala

```diff
@@ -139,6 +138,11 @@ abstract class ActorFlowMaterializer extends FlowMaterializer {
    */
   private[akka] def actorOf(props: Props, name: String): ActorRef
 
+  /**
+   * INTERNAL API
+   */
+  private[akka] def system: ActorSystem
```

Comment:
For including guardian: https://github.com/akka/akka/issues/17298
For using preStart: https://github.com/akka/akka/issues/17299